module2/src/com/epam/onedimensionalarrays/TaskLauncher.java

```diff
@@ -2,6 +2,9 @@
 
 public class TaskLauncher {
     public static void main(String[] args) {
-        ArraySumm.sumElements();
+        ArrayWorker worker = new ArrayWorker();
```

Comment:
You don't need to create the object of this class because it is expected just to have static methods
```suggestion
```